src/styles/isomer-cms/elements/base.scss

```diff
@@ -66,10 +66,6 @@ h6 {
   letter-spacing: 0 !important;
```

Comment:
is there a figma for reference? or a test plan on the PR description

most of it seem to be class changes so maybe its good to verify this visually?

Reply:
Hmm no figma! Just that the text styles should match the other ones